site/ts/grader-submission.ts

```diff
@@ -68,6 +69,10 @@ function init(){
     if (prevQRSuffix){
         qrSuffixInput.value = prevQRSuffix;
     }
+
+    if (prevScanSetting){
+        useScanIdsCheckBox!.checked = prevScanSetting === 'true';
+    }
```

Comment:
I think I understand what this is doing but it's not clear from looking at the PR description. The PR description makes it seems like you are only fixing QR codes when you are doing more.

Reply:
It looks like the "scan user IDs" checkbox wasn't being loaded if already checked and switching between modes - but I agree would be useful to have in PR description to make sure others know full scope of changes